checkmate/async/celery.py

```diff
@@ -0,0 +1,58 @@
+"""Celery app and configuration."""
```

Comment:
Having things called `celery` can cause some import oddness, so I was trying to avoid the exact word. I'll give it a shot and see if it causes any trouble.

Reply:
After a quick chat about this in Slack one of the suggested names was "async" anyway, so I'll leave this alone for now